aspnetcore/signalr/hubs.md

```diff
@@ -609,6 +612,76 @@ Using `Hub<IChatClient>` enables compile-time checking of the client methods. Th
 > [!NOTE]
 > The `Async` suffix isn't stripped from method names. Unless a client method is defined with `.on('MyMethodAsync')`, don't use `MyMethodAsync` as the name.
 
+## Client results
+
+In addition to making calls to clients, the server can request a result from the client. This requires the server to use `ISingleClientProxy.InvokeAsync` and the client to return a result from their `.On` handler.
+
+There are two ways to use the API on the server, the first is to call `Single(...)` on the `Clients` property in a Hub method.
+```csharp
+public class ChatHub : Hub
+{
+    public async Task<string> WaitForMessage(string connectionId)
+    {
+        var message = await Clients.Single(connectionId).InvokeAsync<string>("GetMessage");
+        return message;
+    }
+}
+```
+
+> [!NOTE]
+> Currently using `InvokeAsync` from a Hub method requires setting the [MaximumParallelInvocationsPerClient](xref:signalr/configuration#configure-server-options) option to a value greater than 1.
```

Comment:
Suggestion: I recommend tracking "currently" by a separate issue to circle around and update it later. Here, the text can just inform the reader.

```suggestion
> Using `InvokeAsync` from a Hub method requires setting the [`MaximumParallelInvocationsPerClient`](xref:signalr/configuration#configure-server-options) option to a value greater than 1.
```